addons/website_project/models/project.py

```diff
@@ -0,0 +1,22 @@
+from odoo import api, fields, models
+
+
+class Project(models.Model):
+    _name = 'project.project'
+    _inherit = ['project.project', 'website.published.mixin']
+
+    def action_view_all_rating(self):
+        """ return the action to see all the rating of the project and activate default filters"""
+        if(self.website_published):
+            return self.open_website_url()
+        else:
+            return super(Project, self).action_view_all_rating()
+
+    def open_website_url(self):
+        """ return the action to see all the rating of the project, and activate default filters """
+        return {
+            'type': 'ir.actions.act_url',
+            'name': "Redirect to the Website Project Rating Page",
+            'target': 'self',
+            'url': "/project/rating/%s" % (self.id,)
```

Comment:
You could use ``slug`` here for a beautiful url ;)

Reply:
Done

Reply:
Done, but I kept the old route as well, for compatibility purpose.